infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/LogReplicationEventMetadata.java

```diff
@@ -20,12 +20,15 @@
      */
     private UUID requestId;
 
-    /* Represents the last synced timestamp.
-     *
-     * For snapshot sync, it represents the base snapshot.
-     * For log entry sync, it represents the last log entry synced.
-    */
-    private long syncTimestamp;
+    /*
+     * Represents the last log entry synced timestamp.
+     */
+    private long lastLogEntrySyncedTimestamp;
```

Comment:
does this only apply to logEntrySync?  because for snapshot sync we only ack after all entries are received...

Reply:
correct, it only applies for LogEntrySync.